index.html

```diff
@@ -1150,98 +1155,97 @@ <h2>The <dfn>InteractionOutput</dfn> interface</h2>
                   |itemschema|.
-                  If this throws, re-throw that exception and abort these steps.
+                  If this throws, re-throw that exception and stop.
                 </li>
                 <li>
-                  Set |data|'s [[\value]] to |source|.
+                  Set |data|.[[\value]] to |source|.
                 </li>
               </ol>
             </li>
             <li>
               If |type| is `"object"`, run these sub-steps:
               <ol>
                 <li>
-                  If |source| is not an object, throw {{TypeError}} and abort these steps.
+                  If |source| is not an object, throw {{TypeError}} and stop.
                 </li>
                 <li>
-                  If |schema|'s |properties| is not an object,
-                  throw {{TypeError}} and abort these steps.
+                  If |schema|.|properties| is not an object,
+                  throw {{TypeError}} and stop.
                 </li>
                 <li>
-                  For each property |key| in |source|,
+                  For each |key| in |source|,
                   <ol>
-                    <li>Let |value| be the value of |key|.</li>
-                    <li>Let |propschema| be the value of |key| in |properties|.</li>
+                    <li>Let |value| be |source|[|key|].</li>
+                    <li>Let |propschema| be |properties|.|interactions|[|key|].</li>
                     <li>
                       Let |value| be the result of running the
                       <a>create interaction request</a> steps on |value|, |form|
                       and |propschema|.
-                      If this throws, re-throw that exception and abort these steps.
+                      If this throws, re-throw that exception and stop.
                     </li>
                   </ol>
                 </li>
                 <li>
-                  If |schema|'s |required| is an array, for each |item| in |required|
+                  If |schema|.|required| is an array, for each |item| in |required|
                   check if |item| is a property name in |source|. If an |item| is
-                  not found in |source|, throw {{SyntaxError}} and abort these steps.
+                  not found in |source|, throw {{SyntaxError}} and stop.
                 </li>
                 <li>
-                  Set |data|'s [[\value]] to |source|.
+                  Set |data|.[[\value]] to |source|.
                 </li>
               </ol>
             </li>
           </ol>
         </li>
         <li>
-          Set |idata|'s |data| to a new {{ReadableStream}} created from |idata|'s
-          [[\value]] <a>internal slot</a> as its
-          <a data-cite="streams#underlying-source">
-          underlying source</a>.
+          Set |idata|.|data| to a new {{ReadableStream}} created from
+          |idata|.[[\value]] <a>internal slot</a> as its
```

Comment:
This should be updated using the new internal slot sintax

Reply:
That belongs to a previous PR, not this one, but yes I will correct it.